DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs

```diff
@@ -30,11 +32,15 @@ public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssess
             if (postLearningAssessment.PostLearningAttempts == 0)
             {
                 AssessmentStatus = "Not attempted";
+                StartButtonText = "Start assessment";
+                StartButtonAdditionalStyling = "";
             }
             else
             {
                 AssessmentStatus = GetPassStatus(postLearningAssessment);
                 ScoreInformation = GetScoreInformation(postLearningAssessment);
+                StartButtonText = "Restart assessment";
+                StartButtonAdditionalStyling = "nhsuk-button--secondary";
```

Comment:
I would probably assign to these outside the if, using a ternary statement. However I don't think one is really better than the other, as using ternaries means you'll do the same check several times. So up to you, feel free to keep as it is if you prefer 🙂

Reply:
Yep, sure! Changed in 5bf55d4 🙂